core/src/it/scala/org/ensime/fixture/AnalyzerFixture.scala

```diff
@@ -17,7 +17,17 @@ object AnalyzerFixture {
   private[fixture] def create(search: SearchService)(implicit system: ActorSystem, config: EnsimeConfig, vfs: EnsimeVFS): TestActorRef[Analyzer] = {
     val indexer = TestProbe()
     val projectActor = TestProbe()
-    TestActorRef(Analyzer(projectActor.ref, indexer.ref, search))
+    TestActorRef(Analyzer(projectActor.ref, indexer.ref, search, EnsimeProject(
+      EnsimeProjectId("testProject", "compile"),
+      Seq(),
+      config.projects.flatMap(_.sources).toSet,
+      config.projects.flatMap(_.targets).toSet,
+      List("-encoding", "UTF-16", "-feature", "-deprecation", "-language:postfixOps", "-language:implicitConversions", "-Xlint:-unused,_", "-Yno-adapted-args", "-Ywarn-dead-code", "-Xfuture", "-Ywarn-unused:implicits,imports,-locals,-params,-patvars,-privates", "-Xlint", "-Ywarn-numeric-widen", "-Ywarn-unused-import", "-Ymacro-expand:discard"),
```

Comment:
Actually I did this because we'll be using `scalacOptions` instead of `compiler-args`.
I either had to change a few things in `RefactoringHandlerSpec` or do this here.